pandas/tests/extension/decimal/array.py

```diff
@@ -59,14 +61,28 @@ def __init__(self, values, dtype=None, copy=False, context=None):
         values = np.asarray(values, dtype=object)
 
         self._data = values
-        # Some aliases for common attribute names to ensure pandas supports
-        # these
-        self._items = self.data = self._data
-        # those aliases are currently not working due to assumptions
-        # in internal code (GH-20735)
-        # self._values = self.values = self.data
         self._dtype = DecimalDtype(context)
 
+    # aliases for common attribute names, to ensure pandas supports these
```

Comment:
Is this related to the rounding changes?

Reply:
This issue helped me write a bug *while* working on the rounding changes, and I fixed it to protect myself. does that count as related?